test/lib/classify_test.rb

```diff
@@ -159,81 +159,15 @@ def test_underline
-      assert_generated_class("text-gray-dark",     { color: :gray_dark })
-      assert_generated_class("text-green",         { color: :green })
-      assert_generated_class("text-orange",        { color: :orange })
-      assert_generated_class("text-orange-light",  { color: :orange_light })
-      assert_generated_class("text-purple",        { color: :purple })
-      assert_generated_class("text-pink",          { color: :pink })
-      assert_generated_class("text-white",         { color: :white })
-      assert_generated_class("text-inherit",       { color: :inherit })
-
-      assert_generated_class("color-blue-5",       { color: :blue_5 })
-      assert_generated_class("color-gray-9",       { color: :gray_9 })
-      assert_generated_class("color-purple-3",     { color: :purple_3 })
-
-      assert_generated_class("color-text-primary",   { color: :text_primary })
-      assert_generated_class("color-text-secondary", { color: :text_secondary })
-      assert_generated_class("color-text-tertiary",  { color: :text_tertiary })
-      assert_generated_class("color-text-link",      { color: :text_link })
-      assert_generated_class("color-text-success",   { color: :text_success })
-      assert_generated_class("color-text-warning",   { color: :text_warning })
-      assert_generated_class("color-text-danger",    { color: :text_danger })
-      assert_generated_class("color-text-white",     { color: :text_white })
-      assert_generated_class("color-text-inverse",   { color: :text_inverse })
-
-      assert_generated_class("color-icon-primary",   { color: :icon_primary })
-      assert_generated_class("color-icon-secondary", { color: :icon_secondary })
-      assert_generated_class("color-icon-tertiary",  { color: :icon_tertiary })
-      assert_generated_class("color-icon-info",      { color: :icon_info })
-      assert_generated_class("color-icon-success",   { color: :icon_success })
-      assert_generated_class("color-icon-warning",   { color: :icon_warning })
-      assert_generated_class("color-icon-danger",    { color: :icon_danger })
-    end
-  end
-
-  def test_color_enforcing_functional_colors
-    assert_generated_class("text-orange",        { color: :orange })
-    assert_generated_class("text-orange-light",  { color: :orange_light })
-    assert_generated_class("text-purple",        { color: :purple })
-    assert_generated_class("text-pink",          { color: :pink })
-    assert_generated_class("text-inherit",       { color: :inherit })
-
-    assert_generated_class("color-blue-5",       { color: :blue_5 })
-    assert_generated_class("color-gray-9",       { color: :gray_9 })
-    assert_generated_class("color-purple-3",     { color: :purple_3 })
-
     assert_generated_class("color-text-inverse",   { color: :text_inverse })
-
     assert_generated_class("color-text-primary",   { color: :text_primary })
-    assert_generated_class("color-text-primary",   { color: :gray_dark })
```

Comment:
So just to be clear, we're removing this fallback?

Reply:
The fallbacks will still work because of this change https://github.com/primer/view_components/pull/759/files#diff-03a91cc7818b023b5a2caa70ded0df6ebbb735a0cd9c3187c487c5d568e9a7beR178

The plan is:

1. Deprecate argument in linter
2. Ship linter change out to dotcom
3. remove support from view components
4. Move old classes to dotcom from primer/css
5. Cleanup classes and delete old css

Reply:
Makes sense. If the fallbacks still work, can we keep this test in place?

Reply:
They don't